packages/ia-components/sandbox/theatres/components/audio-player/audio-player-main.jsx

```diff
@@ -98,22 +113,57 @@ export default class TheatreAudioPlayer extends Component {
    * Render function - create tabs that live under the main content area
    */
   createTabs() {
-    const { customSourceLabel } = this.props;
+    const { customSourceLabels } = this.props;
+    const { mediaSource } = this.state;
     const sourceLabel = {
       value: 'player',
-      label: customSourceLabel,
+      label: customSourceLabels.player,
+    };
+
+    const linerNotes = {
+      value: 'liner-notes',
+      label: customSourceLabels.linerNotes,
     };
 
     return (
       <HorizontalRadioGroup
-        options={[sourceLabel]}
+        options={[sourceLabel, linerNotes]}
         onChange={this.toggleMediaSource}
-        selectedValue="player"
+        selectedValue={mediaSource}
         wrapperStyle="tab-bottom"
       />
     );
   }
 
+  /**
+   * Render function - determines whether or not we render liner notes
+   */
+  showLinerNotes() {
+    const { linerNotes } = this.props;
+    const { mediaSource } = this.state;
+
+    const bookReaderStyle = mediaSource === 'liner-notes'
+      ? {
+        visibility: 'visible',
+        position: 'absolute',
+        top: 0,
+        bottom: 0,
+        left: 0,
+        right: 0,
```

Comment:
This is probably where you'll want to add a background color to prevent summary bleed through